dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -308,4 +308,14 @@ public final Response getContentTypes(@Context final HttpServletRequest request,
 		return response;
 	}
 
+	private String getOrderByRealName(String orderbyParam) {
+		if (orderbyParam.equals("modDate")){
+			return "mod_date";
+		}else if (orderbyParam.equals("variable")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "variable" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)